motr/client.h

```diff
@@ -591,6 +591,24 @@ enum m0_entity_type {
 	M0_ET_IDX
 } M0_XCA_ENUM;
 
+/**
+ * Flags passed to m0_entitiy_create(), m0_entity_open() to specify applications's 
+ * behaviour.
+ */
+ enum m0_entity_flags {
+	/**
+	 * During create if this flag is set, thant means application has
+	 * capability to store meta-data and hence pver and lid will be
+	 * returned to application to store.
+	 */
+	M0_ENF_META = 1 << 0,
+	/**
+	 * If this flags is set during entity_create() that means application
+	 * do not support update operation.
+	 */
+	M0_ENF_NO_RMW =  1 << 1
```

Comment:
This is not used yet? Please add comments here.

Can you give examples in comments about how to use the "M0_ENF_META" flag?
Some pseudo is helpful to explain the sequences of getting the pool version and lid, and then pass it to further calling.

Reply:
Updated comment summary.

Reply:
M0_ENF_NO_RMW is not used?

Reply:
OK